cartography/intel/aws/ec2/instances.py

```diff
@@ -408,6 +413,7 @@ def _load_ec2_instance_ebs_tx(
         query,
         ebs_mappings_list=ebs_data,
         update_tag=update_tag,
+        region="global",
```

Comment:
let's use the region from the ec2 instance which which this ebs volume is connected.